examples/voting/src/lib.rs

```diff
@@ -0,0 +1,415 @@
+//! A voting smart contract example.
+//!
+//! # Description
+//! A contract that allows for conducting an election with several voting
+//! options. An end_time is set when the election is initialized. Only accounts
```

Comment:
Perhaps
```suggestion
//! options. An end time is set when the election is initialized. Only accounts
```
Or
Perhaps
```suggestion
//! options. An `end_time` is set when the election is initialized. Only accounts
```

And if you apply this change, please also do it for the `end_time`s in the rest of this module doc.